sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Release History
 
-## 1.2.3 (Unreleased)
+## 1.2.3 (2021-02-09)
 
 - Fixed Azure Stack support for the NodeJS version of the `InteractiveBrowserCredential`.
 - The 'keytar' dependency has been updated to the latest version.
+- Now this library won't be overriding the global Axios defaults. This has been possible both through an update of the library and an update of the `@azure/msal-node` dependency. Fixes issue [13343](https://github.com/Azure/azure-sdk-for-js/issues/13343).
```

Comment:
For the first sentence, I might instead say "No longer overrides global Axios defaults. ..."

Reply:
Sounds good!